Jamie/survival fix stratified output

charlesknightsbridge opened the pull request:
Output was missing the degrees of freedom for the log rank test.
Comments added for possible future revision of this test

charlesknightsbridge committed:
Comments only

charlesknightsbridge committed:
Add degrees of freedom to log rank test

charlesknightsbridge committed:
Correct formatting of log rank chi-square

torstenchr commented:
Is this PR ready on your side? There are still some TODO's in the commit - is that on purpose?

charlesknightsbridge committed:
Removed unnecesary code

There is still one TODO; it's not clear from classic EpiData or other sources what to do, so output will be zero when there is a single at risk subject at the last time interval.

charlesknightsbridge commented:
> Is this PR ready on your side? There are still some TODO's in the commit - is that on purpose?

One TODO done. The other is just a reminder for the future.
